src/node.cc

```diff
@@ -185,7 +185,8 @@ static int v8_thread_pool_size = v8_default_thread_pool_size;
 static bool prof_process = false;
 static bool v8_is_profiling = false;
 static bool node_is_initialized = false;
-static node_module* modpending;
+static uv_once_t init_once = UV_ONCE_INIT;
```

Comment:
The name of this variable should indicate that it refers to `thread_local_modpending` – so, like, maybe `init_modpending_once`?